From 4711fc19b6b83d572fd6eafd64a834559cae8ce2 Mon Sep 17 00:00:00 2001
From: Hannes Reinecke <hare@suse.de>
Date: Wed, 26 Feb 2014 12:50:09 +0100
Subject: [PATCH 2/2] Install examples in documentation directory

The examples should be installed in the documentation directory,
not the standard configuration directory.

Signed-off-by: Dirk Mueller <dmueller@suse.com>
Acked-by: Hannes Reinecke <hare@suse.de>
---
 conf/Makefile     | 6 ++++--
 conf/targets.conf | 5 +++++
 2 files changed, 9 insertions(+), 2 deletions(-)

diff --git a/conf/Makefile b/conf/Makefile
index bff222f..abb763e 100644
--- a/conf/Makefile
+++ b/conf/Makefile
@@ -1,4 +1,5 @@
 sysconfdir ?= /etc
+docdatadir ?= /usr/share/doc/packages/
 
 EXAMPLES = targets.conf.example targets.conf.vtl.L700 targets.conf.vtl.MSL2024
 
@@ -11,9 +12,10 @@ install:
 	if [ ! -f $(DESTDIR)$(sysconfdir)/tgt/targets.conf ] ; then \
 		install -m 644 targets.conf $(DESTDIR)$(sysconfdir)/tgt ; \
 	fi
-	install -d -m 755 $(DESTDIR)$(sysconfdir)/tgt/examples
+	install -d -m 755 $(DESTDIR)$(sysconfdir)/tgt/conf.d
+	install -d -m 755 $(DESTDIR)$(docdatadir)/tgt/examples
 	for f in $(EXAMPLES) ; do \
-		install -m 644 examples/$$f $(DESTDIR)$(sysconfdir)/tgt/examples ;\
+		install -m 644 examples/$$f $(DESTDIR)$(docdatadir)/tgt/examples ;\
 	done
 	install -d $(DESTDIR)$(sysconfdir)/tgt/conf.d
 
diff --git a/conf/targets.conf b/conf/targets.conf
index 1bb4435..70b608a 100644
--- a/conf/targets.conf
+++ b/conf/targets.conf
@@ -1,5 +1,10 @@
 # Empty targets configuration file -- please see the package
 # documentation directory for an example.
+
+#
+# Include configuration from directory
+# 
+include /etc/tgt/conf.d/*.conf
 #
 # You can drop individual config snippets into /etc/tgt/conf.d
 include /etc/tgt/conf.d/*.conf
-- 
1.8.1.4

